Tools/TestWebKitAPI/Tests/WebKitCocoa/WebsitePolicies.mm

```diff
@@ -1754,3 +1754,15 @@ - (void)webView:(WKWebView *)webView runJavaScriptAlertPanelWithMessage:(NSStrin
     [webView loadTestPageNamed:@"color-scheme"];
     [webView waitForMessage:@"dark-detected"];
 }
+
+TEST(WebpagePreferences, ToggleNetworkConnectionIntegrity)
+{
+    auto preferences = adoptNS([WKWebpagePreferences new]);
+    EXPECT_FALSE([preferences _networkConnectionIntegrityEnabled]);
+    [preferences _setNetworkConnectionIntegrityEnabled:YES];
```

Comment:
I mean, I guess "a test for every change", but yikes :)

Reply:
😅